### PR TITLE
just one small refactor

### PR DESCRIPTION
**Pull request by dneise**
shorten `_remove_acknowledged_and_old_calls` a bit

### CODE REVIEW

File: shifthelper/tools/__init__.py [hunk 1]
```diff
@@ -30,11 +31,20 @@
...
        wait_exponential_max=1000,  # but wait 1 second per try maximum
        )
+@cached(cache=TTLCache(1, ttl=30))
 def get_alerts():
     alerts = requests.get(config['webservice']['post-url'])
     return alerts.json()
 
 
+def is_alert_acknowledged(uuid):
+    alerts = get_alerts()
```
Comment (MaxNoe) [inline shifthelper/tools/__init__.py]: This should also be in the try..except block, right?

Comment (dneise) [inline shifthelper/tools/__init__.py]: Ah right .. so if we can't even get any alerts ... the answer is still `False`...

File: shifthelper/tools/__init__.py [hunk 2]
```diff
@@ -30,11 +31,20 @@
...
+    try:
+        return alerts[uuid]['acknowledged']
+    except:
```
Comment (dneise) [inline shifthelper/tools/__init__.py]: Ah, but after moving the `get_alerts` into the try block .. I'm not sure which exception can occur... I'll wrap it

**Comment by dneise** [comment]
I think this is not important .. should we close it?

**Comment by MaxNoe** [comment]
yeah

**Pull request closed without merging**